eth-rpc-adapter/src/transports/http.ts

```diff
@@ -59,6 +59,7 @@ export default class HTTPServerTransport extends ServerTransport {
       if (req.body.length > this.options.batch_size) {
         const error = new InvalidRequest();
         result = {
+          id: null,
```

Comment:
```suggestion
          id: req.body?.id ?? null,
```

> If there was an error in detecting the id in the Request object (e.g. Parse error/Invalid Request), it MUST be Null.

from my understanding id should only be `null` when there is no id in the request body （fail to detect）, but I think either way should be fine since I checked infura eth rpc response they don't even have an id for invalid json request……

Reply:
the body is an array, each item has an id, so if I don't know which one, I use the `null`.

```
[
    {
        "id": 16,
        "jsonrpc": "2.0",
        "method": "eth_getBalance",
        "params": [
            "0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1",
            "latest"
        ]
    },
    {
        "id": 226,
        "jsonrpc": "2.0",
        "method": "eth_getBalance",
        "params": [
            "0x7dda0dc1c85bfe07f959fb6ed59be8486f48afc5",
            "0xB0EE4"
        ]
    }
]
```